packages/lib/storage/service.ts

```diff
@@ -188,6 +188,14 @@ export const getUploadSignedUrl = async (
   if (!IS_S3_CONFIGURED) {
     try {
       const { signature, timestamp, uuid } = generateLocalSignedUrl(updatedFileName, environmentId, fileType);
+      let fileUrl = "";
+
+      // self-hosted and no S3
+      if (!IS_FORMBRICKS_CLOUD) {
```

Comment:
What if a self-hosted user is using S3?
Wouldn't `IS_S3_CONFIGURED` make more sense here?

Reply:
yeah this wouldn't work with users that are self hosting and have s3 configured, the next/image component does not load the image from S3 with a relative url 😞